ros_buildfarm/scripts/doc/build_rosdoc2.py

```diff
@@ -64,7 +63,7 @@ def main(argv=sys.argv[1:]):
             env['PATH'] = ''
         else:
             env['PATH'] += ':'
-        env['PATH'] += '/home/buildfarm/.local/bin'
+        env['PATH'] += os.path.join(env['HOME'], '.local/bin')
```

Comment:
This was creating problems for my nuclearsandwich/uid-override patches which are required if your host user id is `1000` as the ubuntu:noble image now has an `ubuntu` user there.